wowchemy/archetypes/authors/_index.md

```diff
@@ -6,6 +6,9 @@ title: "{{ replace .Name "-" " " | title }}"
 authors:
 - "{{ urlize .Name }}"
 
+aliases:
```

Comment:
Isn't aliases a reserved option by Hugo? If so, we'd need to use something else like `nicknames`...

Reply:
Oh, whoops. I see now. Didn't know about those.